ui/job-view/pushes/PushJobs.jsx

```diff
@@ -37,6 +37,18 @@ class PushJobs extends React.Component {
 
   onMouseDown = (ev) => {
     const { togglePinJob } = this.props;
+
+    if (
+      'classList' in ev.target &&
+      ev.target.classList.contains('group-select-all-runnable')
+    ) {
+      const groupNode = ev.target.parentNode;
+      if (groupNode.classList.contains('job-group')) {
+        this.handleGroupSelectAllRunnableClick(ev, groupNode);
+      }
+      return;
+    }
+
```

Comment:
This could be done in `JobGroup.jsx` just by adding `onclick` handler to the `<button>` there, but I followed existing per-job handling here.